app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -252,17 +252,9 @@ class SamplesView extends React.Component {
   };
 
   renderCollectionTrigger = () => {
-    const {
-      currentDisplay,
-      mapPreviewedSamples,
-      samples,
-      selectedSampleIds,
-    } = this.props;
+    const { samples, selectedSampleIds } = this.props;
 
-    // NOTE(jsheu): For mapSidebar sample names to appear in CollectionModal,
-    // they need to be presently loaded/fetched. Otherwise the ids work but says "and more..." for un-fetched samples.
-    const targetSamples =
-      currentDisplay === "map" ? mapPreviewedSamples : samples.loaded;
+    const targetSamples = samples.loaded;
```

Comment:
This isn't removing a feature where the user selects some samples on the map and then goes to the sample list? Just making sure. Don't know the code.

Reply:
No, this samples for the sidebar were loaded independently from the samples for the main table. The last changes changed that, and all the data comes from the same collection now.